astaire/astaire.supervisord.conf

```diff
@@ -0,0 +1,5 @@
+[program:astaire]
+command=/etc/init.d/astaire run
+startretries=100
```

Comment:
Yes -- this is because Astaire repeatedly dies if cluster_settings doesn't exist, and it exceeds the maximum permitted number of retries before cluster manager has created it.   I'll raise an issue against Astaire, but unless it causes an issue somewhere I suspect we won't fix it in a hurry?